polyply/src/simple_seq_parsers.py

```diff
@@ -118,15 +140,18 @@ def _parse_plain(lines, DNA=False, RNA=False):
                 resname = ONE_LETTER_DNA[token]
             elif token in ONE_LETTER_RNA and RNA:
                 resname = ONE_LETTER_RNA[token]
+            elif token in ONE_LETTER_AA and AA:
+                resname = ONE_LETTER_AA[token]
             else:
                 msg = f"Cannot find one letter residue match for {token}"
                 raise IOError(msg)
 
             monomers.append(resname)
 
     # make sure to set the defaults for the DNA and RNA terminals
-    monomers[0] = monomers[0] + "5"
-    monomers[-1] = monomers[-1] + "3"
+    if RNA or DNA:
+        monomers[0] = monomers[0] + "5"
+        monomers[-1] = monomers[-1] + "3"
```

Comment:
do we need any terminal patching for the proteins? I think since so far we only support Martini we don't need it but something to think about.

Reply:
Agreed for case if we're just dealing with Martini - easy enough to pass terminal patching from another .ff file anyway, but something to consider I guess